server/testutils/data/handoverData.ts

```diff
@@ -20,7 +20,11 @@ const testHandoverContext: HandoverContextData = {
     sexuallyMotivatedOffenceHistory: 'YES',
   },
   assessmentContext: { oasysAssessmentPk: '306708', assessmentVersion: 0 },
-  sentencePlanContext: { oasysAssessmentPk: '306708', planVersion: 0 },
+  sentencePlanContext: {
+    oasysAssessmentPk: '306708',
+    planId: 'afgo3cacf-7160-4431-9b4d-c560be9c9f5c',
```

Comment:
`g`? valid uuid's have a-f, 0-9

Reply:
just for you bud https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/157/commits/2d682f051dd945cd00daed6cc6abeeb1ec68d1fc